src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsBaseObject.java

```diff
@@ -601,6 +606,31 @@ public AdsBaseObject triggerAdSlot(String slotName) {
     return this;
   }
 
+  private void triggerFMR() {
+    final By FMRSelector = By.cssSelector(AdsContent.getSlotSelector(AdsContent.FLOATING_MEDREC));
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Rename this local variable name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS00117)